code/demos/Simulation.ipynb

```diff
@@ -672,14 +682,30 @@
    "cell_type": "code",
    "execution_count": null,
    "metadata": {
-    "collapsed": false
+    "collapsed": true
    },
    "outputs": [],
    "source": [
     "# these don't have to be normalized\n",
-    "ln_likelihoods = reg_vals(sheet_cake + ln_pzs[:, np.newaxis, :, np.newaxis])\n",
-    "\n",
-    "# look at how much narrower they are! this should be visible in all types, but the color scaling is bad for Ibc & II\n",
+    "ln_likelihoods = reg_vals(sheet_cake + ln_pzs[:, np.newaxis, :, np.newaxis])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We plot the combined likelihoods for when lightcurves and host galaxy photometry are available.  For the same supernovae as in the previous plot, you can see how the constraints in $z$ become much narrower when th redshift likelihoods are included."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
```

Comment:
See above - what are people "getting" our of the plots and how can we show that most effectively. Let's discuss later

Reply:
Re: plots, the plots in the notebook are for us (at least me) to check that our log-probabilities look the way they should, at the level of debugging, but I don't think all (or even possibly any) of them should be in the paper.